src/runtime_src/core/pcie/linux/CMakeLists.txt

```diff
@@ -71,6 +73,7 @@ target_link_libraries(xrt_core_static
   dl
   rt
   pthread
+  Boost::boost
```

Comment:
ditto.

Reply:
I hear you about std::any, and for sure we'd prefer that, unfortunately C++17 is not yet universally supported on all the platforms we need to build against.  But maybe we should consider some conditionally magic, at least that would resolve it for your use-case and we can then punt on the INTERFACE changes?

Reply:
Hmm...If there are C++ 17 issues, shouldn't the XRT-Pipeline catch them for it is my understanding that each supported platform is built.